x/registration/remote_attestation/remote_attestation.go

```diff
@@ -184,7 +184,6 @@ func verifyAttReport(attnReportRaw []byte, pubK []byte) ([]byte, error) {
 				if err != nil && len(platInfo) != 105 {
 					return nil, errors.New("illegal PlatformInfoBlob")
 				}
-				platInfo = platInfo[4:]
```

Comment:
Why?

Reply:
never used.

I questioned it too, an d checked, and all the things seemed ok?

wdyt?

Reply:
Yeah an oversight probably :confused: